orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/load_table.py

```diff
@@ -29,26 +29,28 @@ def _diff_hca_table(
 
     query = f"""
     SELECT J.datarepo_row_id, S.*, {datarepo_key}
-    FROM {file_metadata_type} S FULL JOIN {target_hca_dataset.project_id}.{fq_dataset_id}.{file_metadata_type} J
+    FROM {metadata_type} S FULL JOIN {target_hca_dataset.project_id}.{fq_dataset_id}.{metadata_type} J
     USING ({primary_key})
     """
     destination = f"{scratch_dataset_name}.{joined_table_name}"
+
+    metadata_path = "file-metadata-with-ids" if is_file_metadata else "metadata"
```

Comment:
here is the switch for file vs nonfile metadata external source paths

Reply:
Hmm I'm not sure about the rename; the dataflow pipeline writes the `metadata` output, which includes the file metadata _without_ file ids as well as the rest of the metadata entities, and it seems like pointless processing to move the non-file metadata to another folder and name it differently